docs/api/sso-saml.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "SSO saml"
+summary: |
+  `/api/saml` endpoints.
+---
+
+# SSO saml
```

Comment:
```suggestion
# SSO SAML
```

Is it possible to make the change here? Or is the title picked up from somewhere else via the script?

Reply:
We'll have to make the change in the code.